content/articles/where-incorporate-eu-citizen.md

```diff
@@ -0,0 +1,157 @@
+<!-- PELICAN_END_SUMMARY -->
+
+## Our requirements
+As mentioned in the introduction, we are French and German and the only common language is English so being able to do all (or at least the vast majority) government/bank interactions in English is a hard requirement.
+
+Since I personally like to travel and moving back to France/Germany is always a possibility, we also need to be able to incorporate in a country without living there. This is especially true if you are planning to be a digital nomad.
+
+Lastly, we want a business-friendly country with a good record on privacy and surveillance law as we run [Passopolis](https://passopolis.com/), an open-source password manager that relies on end-to-end encryption.
+
+To sum it up, our requirements are:
+
+- we need to be able to do things in English
+- we need to be able to incorporate without living there
+- we want its government to be privacy friendly and not authoritarian
+
+Your own requirements will obviously differ and some countries that we disregard in the next section might be the perfect fit for you. Or you might want to get out of the EU to not deal with [VAT MOSS](https://blog.wearewizards.io/looking-at-payments-solution-for-saas-in-europe) like [Ghost](https://blog.ghost.org/moving-to-singapore/) for example.
+
+## Countries selection
+
+France, Germany, Spain, Portugal, the Netherlands and Denmark fail as I couldn't find all the needed information on incorporation in English and the local language is required for all interactions.
+
+The US (via [Stripe Atlas](https://stripe.com/atlas)) also doesn't make much sense from a EU citizen point of view as we have access to better choices as we will see in a moment and opens you to stupid patent laws.
+This might be different if you are planning to raise money in the US though.
+
+With the rejected countries out of the way, let's go over the potential choices I have identified after a bit of research.
+
+To keep the each country information DRY (Don't Repeat Yourself), one advice common to every country including your own is to get an accountant from the start as they are not that expensive in the grand scheme of things and let you worry only about your business.
+
+Do note that you can Google "company formation country_X" and you will get lots of companies offering services helping you with incorporation and taxes.
+
+### United Kingdom
+#### Trivia
+The UK is very business friendly and the biggest tech centre in Europe. You can probably go to a tech meetup every day if you wanted to and you get to meet and work with people from all over the world.
+
+In terms of privacy laws, the UK is not doing too great with the [IP bill](https://en.wikipedia.org/wiki/Investigatory_Powers_Bill) it's trying to pass.
+
+The obvious elephant in the room is **Brexit**. No one really knows when it will actually happen and what will be the outcome so there is a great deal of uncertainty about, well, everything.
```

Comment:
2 years deadline from the moment they activate it, and we don't know when that will happen